provisioner/terraform/inittimings.go

```diff
@@ -0,0 +1,145 @@
+package terraform
+
+import (
+	"slices"
+	"time"
+
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/provisionersdk/proto"
+)
+
+var (
+	// resourceName maps init message codes to human-readable resource names.
+	// This is purely for better readability in the timing spans.
+	resourceName = map[initMessageCode]string{
+		initInitializingBackendMessage:    "Backend",
```

Comment:
Can you stick to the existing naming convention which is all lowercase?
Also I think we need to be more explicit about the stages. What does "Backend" mean?

Reply:
I want to spend time on the build timeline in the UI more. I would like it to also go 1 more hierarchical depth. 

Will defer to future work on that though